config/routes.rb

```diff
@@ -13,8 +13,11 @@
     end
   end
 
+
   namespace :audits do
-    get '/', to: "audits#index"
+    get '/', to: "allocations#index"
+    get '/my_content', to: "audits#index", as: 'my_content'
+    get '/all_content', to: "allocations#index", as: 'all_content'
```

Comment:
I'm not sure about having two routes leading to `allocations#index`. We should do one of:

- remove the `/all_content` endpoint altogether
- redirect the root path: `get "/", to: redirect("/all_content")`

Reply:
👍